components/otelopscol/receiver/dcgmreceiver/go.mod

```diff
@@ -6,23 +6,23 @@ require (
 	github.com/NVIDIA/go-dcgm v0.0.0-20240910155525-85ceb314ca65
 	github.com/google/go-cmp v0.7.0
 	github.com/stretchr/testify v1.10.0
-	go.opentelemetry.io/collector/component v1.27.0
-	go.opentelemetry.io/collector/component/componenttest v0.121.0
-	go.opentelemetry.io/collector/config/confignet v1.27.0
-	go.opentelemetry.io/collector/confmap v1.27.0
-	go.opentelemetry.io/collector/consumer v1.27.0
-	go.opentelemetry.io/collector/consumer/consumertest v0.121.0
-	go.opentelemetry.io/collector/filter v0.121.0
-	go.opentelemetry.io/collector/pdata v1.27.0
-	go.opentelemetry.io/collector/receiver v0.121.0
-	go.opentelemetry.io/collector/receiver/receivertest v0.121.0
-	go.opentelemetry.io/collector/scraper v0.121.0
-	go.opentelemetry.io/collector/scraper/scraperhelper v0.121.0
+	go.opentelemetry.io/collector/component v1.32.0
+	go.opentelemetry.io/collector/component/componenttest v0.126.0
+	go.opentelemetry.io/collector/config/confignet v1.32.0
+	go.opentelemetry.io/collector/confmap v1.32.0
+	go.opentelemetry.io/collector/consumer v1.32.0
+	go.opentelemetry.io/collector/consumer/consumertest v0.126.0
+	go.opentelemetry.io/collector/filter v0.126.0
+	go.opentelemetry.io/collector/pdata v1.32.0
+	go.opentelemetry.io/collector/receiver v1.32.0
```

Comment:
@braydonk   The `go.opentelemetry.io/collector/receiver` version is updated to use the core version (used contrib version previously). Is this ok? This change is auto generated

Reply:
The make target that updates dependencies uses a file from the collector repo to detect stability of components: https://github.com/open-telemetry/opentelemetry-collector/blob/main/versions.yaml

It correctly determined that this component is now stable, so this is correct.